nft/macros/nft_wash_trades.sql

```diff
@@ -112,14 +112,17 @@ WITH filter_1 AS (
     SELECT unique_trade_id
     , true AS flashloan
     FROM {{ ref('nft_trades') }} nftt
-    INNER JOIN {{ source('dex','flashloans') }} df ON df.blockchain='{{blockchain}}'
-        AND df.block_time=nftt.block_time
+    INNER JOIN {{ source('lending','flashloans') }} df ON df.blockchain='{{blockchain}}'
+        AND df.block_number=nftt.block_number
```

Comment:
another good rule of thumb:
use the partition column in join conditions. both of these tables use `block_month`. if you add that as an extra condition, the query engine will have less files to scan through. then can further filter on that small subset by `block_number`, `tx_hash`

Reply:
i think it's okay to proceed for now as there isn't a performance concern in current table sizes. something for us to track when finalizing a refactor on `nft.wash_trades` (once we complete `nft.trades`)